src/main/kotlin/com/matthewrussell/trchunkbrowser/app/mainview/MainView.kt

```diff
@@ -25,9 +26,31 @@ class MainView : View() {
     override val root = stackpane {
         vbox {
             addClass(MainViewStyles.root)
-            add(JFXSnackbar(this).apply {
+            add(JFXSnackbar(this).apply { // Progress snackbar
+                viewModel.snackBarProgress.subscribe { progress ->
+                    if(progress < 100) {
+                        enqueue(JFXSnackbar.SnackbarEvent(
+                            messages["export_in_progress"],
+                            "",
+                            0,
+                            true,
+                            null))
+                    } else {
+                        this.visibleProperty()
+                            .toObservable()
+                            .takeUntil { it }
+                            .doOnComplete {
+                                this.close()
+                            }
+                            .subscribe()
+                    }
+                }
+            })
+            add(JFXSnackbar(this).apply { // Messages snackbar
                 viewModel.snackBarMessages.subscribe {
-                    enqueue(JFXSnackbar.SnackbarEvent(it))
+                    enqueue(JFXSnackbar.SnackbarEvent(it, "OK", 0, true, EventHandler {
+                        this.close()
+                    }))
```

Comment:
formatting

Reply:
@sarabiaj Should I put parameters on their own lines here as well?